app/frontend/src/features/auth/signup/FeedbackForm.test.tsx

```diff
@@ -62,9 +64,15 @@ describe("signup form (feedback part)", () => {
       expect(params[1].getExpertise()).toBe("I have lots of expertise!");
     });
 
-    expect(result.current.authState.authenticated).toBe(false);
-    expect(result.current.authState.flowState).toStrictEqual(
-      stateAfterFeedback
-    );
+    const { result: result2 } = renderHook(() => useAuthContext(), {
+      wrapper,
+    });
```

Comment:
How come we need to render the hook again from scratch? Will using the `rerender` function in the object returned by `renderHook` not work?

Reply:
I tried it, it always gave the same thing unless I did a new one